server/TossupRoom.js

```diff
@@ -185,6 +192,15 @@ class TossupRoom {
         const type = message.type || '';
 
         switch (type) {
+        case 'bot-difficulty':
+            this.settings.botDifficulty = message.value;
```

Comment:
Please add invalidations for what this can be, from what I can tell a user could input a string as a value here.